go.mod

```diff
@@ -109,7 +109,7 @@ require (
 	google.golang.org/grpc v1.27.1
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/square/go-jose.v2 v2.2.2
-	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
this picks up the problematic default line-length change discussed in https://github.com/go-yaml/yaml/pull/670 (seen in the conformance output results, but affecting API serializations as well)

if something is requiring at least this version, we have to go all the way to v2.4.0 to pick up the fix for this regression in https://github.com/go-yaml/yaml/commit/7649d4548cb53a614db133b2a8ac1f31859dda8c

Reply:
OK, didn't realize it was that big a problem.

Do you happen to know if there's a k8s-wide plan here for sticking to 2.2.8 or moving to 2.4.0?  

I'm not sure right now how hard it would be to pin at 2.2.8

Reply:
Moving to 2.4.0 is fine, just nothing had prompted it yet